Directory traversal and VISDOM data source creation

bricof opened the pull request:
* Added directory traversal to the p39toCSV.py file, and changed the sample_data folder to use the prop39schools directory structure

* Added headers to the _INTERVAL.csv files

* Added the utility provider name as a column in the _BILL.csv files

* Added a notebook showing how to use the resulting collection of csv files to produce the necessary tables for VISDOM in a sql database

Commit:
changed gitignore to include DS_Store garbage

Commit:
changed sample data to mirror PGE directory structure, and changed script to walk through it

Commit:
added notebook to go from csv files to sql

Commit:
added a check to make sure that the 'agreement' field is present

Commit:
minor - changed default paths back to sample data

Commit:
updated csv_to_sql notebook after testing with full data

Commit:
updated notebook for mysql database creation for visdom, including interventions, added script for visdom data source in R

bricofStitchFix commented:
Updated the database creation script (including the addition of an intervention table). Moved it to a VISDOM folder within the notebooks collection and added an R script for the VISDOM data source in there as well. 

It still has a few issues to figure out (e.g. why are some of the meter_data sql inserts failing? can we grab a "Site CDS Code" from the xml to improve the link from the accounts table to the intervention table?), but does produce a functioning VISDOM data source for exploration.

bricof commented:
Updated the database creation script (including the addition of an intervention table). Moved it to a VISDOM folder within the notebooks collection and added an R script for the VISDOM data source in there as well.

It still has a few issues to figure out (e.g. why are some of the meter_data sql inserts failing? can we grab a "Site CDS Code" from the xml to improve the link from the accounts table to the intervention table?), but does produce a functioning VISDOM data source for exploration.
